src/app/@ansyn/ol/mapSourceProviders/open-layers-ESRI-4326-source-provider.ts

```diff
@@ -33,9 +31,9 @@ export class OpenLayerESRI4326SourceProvider extends OpenLayersMapSourceProvider
 				return config.baseUrl
 					.replace('{z}', (tileCoord[0] - 1).toString())
 					.replace('{x}', tileCoord[1].toString())
-					.replace('{y}', (-tileCoord[2] - 1).toString());
+					.replace('{y}', (tileCoord[2]).toString());
 			},
-			wrapX: true
```

Comment:
Why this change?

Reply:
is for not over draw this layer on his x-axis.